src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ExceptionsGenericError.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.Data;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
+{
+    public class ExceptionsGenericErrors : IClassFixture<ExceptionGenericErrorFixture> {
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        [ActiveIssue(10036)]
+        public void TestCommandOptionWithNoTceFeature () {
+            SqlConnectionStringBuilder sb = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+            CertificateUtility.ChangeServerTceSetting (false, sb); // disable TCE on engine.
+            using (SqlConnection conn = CertificateUtility.GetOpenConnection(false, sb, fSuppressAttestation: true))
+            {
+                using (SqlCommand cmd = new SqlCommand(ExceptionGenericErrorFixture.encryptedProcedureName, conn, null, SqlCommandColumnEncryptionSetting.Enabled))
+                {
+                    SqlParameter param = cmd.Parameters.AddWithValue("@c1", 2);
+                    cmd.CommandType = CommandType.StoredProcedure;
+                    string expectedErrorMessage = "SQL Server instance in use does not support column encryption.";
+                    InvalidOperationException e = Assert.Throws<InvalidOperationException>(() => cmd.ExecuteNonQuery());
+                    Assert.Contains(expectedErrorMessage, e.Message);
+                }
+                conn.Close();
+            }
+            // Turn on TCE now
+            CertificateUtility.ChangeServerTceSetting (true, sb); // enable tce
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        public void TestDataAdapterAndEncrytionSetting () {
+            SqlConnectionStringBuilder sb = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+            // Create a new SqlCommand for select and delete
+            SqlConnection conn = CertificateUtility.GetOpenConnection(false, sb);
```

Comment:
Lines 34, 35, 40, 72, 73, 85, 86, and 137 should be using blocks for the SqlConnection/SqlCommand objects. And no need for conn.Close() when using using blocks (lines 24, 66, 129, and 146).